libraries/AP_HAL_ChibiOS/hwdef/PixFlamingo-F767/defaults.parm

```diff
@@ -0,0 +1,4 @@
+RC2_REVERSED 1
+MOT_SAFE_DISARM 1
+BARO_ALTERR_MAX 6000
```

Comment:
this is quite a big number, it might suggest the were not using the correct baro driver?